_scripts/atest.py

```diff
@@ -12,22 +12,31 @@
     # ("Windows", "5.0", "6"): ["--include", "not-supported", "--runemptysuite"]
 }
 
+LAB_MAJOR_ENV_VARS = {
+    1: {"JUPYTER_LIBRARY_APP": "NotebookApp"},
+    2: {"JUPYTER_LIBRARY_APP": "NotebookApp"},
+}
+
 NON_CRITICAL = [
     ## Historically supported nteract_on_jupyter
     # ["client:nteract_on_jupyter"],
 ]
 
 PABOT_DEFAULTS = [
     "--testlevelsplit",
-    "--processes",
-    "4",
+    *("--processes", "4"),
```

Comment:
A nice solution, I've often used `--something=else` to mitigate the line-split, but not all CLI parsers handles the equal sign.